app/services/users/invitation_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Users
+  class InvitationService < ApplicationService
+    def call
+      invite = User.invite!(invite_params) { |u| u.skip_invitation = true }
+      if invite.errors.empty?
+        invite.create_family(family_params.merge(owner: current_user.id))
+        invite.update_attribute(:family_id, invite.family.id)
+        invite.update_attribute(:invited_by_id, current_user.id)
+        message = "#{invite['first_name']} #{invite['last_name']} added you as family\
+        member on FellowDriver. Click the link below to accept the invitation:\
+        http://localhost:3000/api/users/invitation/accept?invitation_token=#{invite.raw_invitation_token}"
+        Twilio::TwilioTextMessenger.perform(message)
+        return OpenStruct.new(success?: true,
+                              data: { invite_token: invite.raw_invitation_token,
+                                      user: invite.present.invite_page_context },
+                              errors: nil)
+      else
+        return OpenStruct.new(success?: false, data: nil, errors: invite.errors)
+      end
+    end
+
+    private
+
+    def invite_params
+      params.permit(:first_name, :last_name, :email, :phone, :skip_invitation)
```

Comment:
params should be permitted in controller

Reply:
done